db/Pos/DB/Rocks/Redirect.hs

```diff
@@ -0,0 +1,54 @@
+{-# LANGUAGE RankNTypes          #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeFamilies        #-}
+
+-- | Redirects enable instances of some type classes.
+-- Here it is related to type classes from 'Pos.DB.Class'.
+
+module Pos.DB.Rocks.Redirect
```

Comment:
You're right, it was used by Vlad as it is. Let's move the implementation into `Functions`.